pkgs/top-level/all-packages.nix

```diff
@@ -676,6 +676,8 @@ with pkgs;
   makeWrapper = makeSetupHook { deps = [ dieHook ]; substitutions = { shell = targetPackages.runtimeShell; }; }
                               ../build-support/setup-hooks/make-wrapper.sh;
 
+  makeWrapperAuto = callPackage ../build-support/setup-hooks/make-wrapper-auto { };
```

Comment:
Is there a better name for this? `makeWrapperAuto` does not really give away what it does.

Reply:
> Is there a better name for this? `makeWrapperAuto` does not really give away what it does.

Is `wrapAuto` better? The goal is to make it replace `wrapQtAppsHook` and `wrapGAppsHook`. I updated the PR so that `combineWrappersInfo` is a passthru function of the `makeWrapperAuto` hook.

Reply:
I think the part that is unclear is `Auto`... What is automatic about it? (I know what is automatic, but the name isn't saying.) Maybe `declareWrapper` is a better name?